CMakeLists.txt

```diff
@@ -266,8 +266,6 @@ IF(HAVE_PY_INTERP)
 			OUTPUT_VARIABLE PYTHON_DEST
 		)
 
-		# Chop off leading /usr
-		STRING(REPLACE "/usr" ${CMAKE_INSTALL_PREFIX} PYTHON_DEST "${PYTHON_DEST}")
```

Comment:
Prefixing `PYTHON_DEST` by `CMAKE_INSTALL_PREFIX` makes sense for a case when one would like to have separate folder to install building results. As make system user I expect if I specify prefix then all files will be copied under this prefix folder, so I would keep prefixing. 

However instead of replacing `/usr` prefix could be specified as a `prefix` argument of the `get_python_lib()` call in line 265: https://github.com/opencog/atomspace/blob/b84086239471008db68d8ee9572643d2fd207a9d/CMakeLists.txt#L265

Separate issue is that standard Python modules looking path doesn't include `/usr/local/...` folder.

Reply:
Excellent idea. Modified. (Also rebased on macos branch to make it easier for me to test, but tested both on mac and ubuntu.)